app/views/assignments/edit/_general.html.erb

```diff
@@ -94,10 +94,6 @@ function autogenerate_submission(){
     <td style='padding:5px' id='assignment_team_assignment_field'>
 
       <div class="form-inline">
-        <%= check_box_tag('team_assignment', 'true', @assignment_form.assignment.team_assignment?, {:onChange => 'hasTeamsChanged()'}) %>
-        <% @assignment_form.assignment.max_team_size %>
```

Comment:
Was this doing something @johnbumgardner?

Reply:
no